src/ansys/fluent/core/quantity.py

```diff
@@ -0,0 +1,184 @@
+from itertools import permutations
+
+import pint
+from pint import Unit
+
+base_units = pint.UnitRegistry(autoconvert_offset_to_baseunit=True)
+base_units.default_system = "SI"
+
+# Unit definitions
+base_units.define("1.e15-particles/kg = 1 kg^-1 = 1^15particles kg^-1")
```

Comment:
@hpohekar I'm not convinced that we should cover this kind of "special" stuff from Fluent. Not only does it seem special, but considering what we can already see being returned with units from the settings API right now:
```
>>> s.setup.named_expressions()
{'parameter_1': {'definition': '1.5 [m/s]', 'description': '', 'input_parameter': False, 'output_parameter': False}}
```
that is not based on Fluent units so those special units will not be seen there, and are not supported there. The proposal is to only use standard units and to at least be self-consistent. Let's eliminate this from the implementation.

Reply:
Yes, sure.

Reply:
Yes, agree with Sean on this one.  Let's keep this module as generic as possible.

Reply:
Done.